chainer/functions/math/clip.py

```diff
@@ -38,7 +38,11 @@ def backward(self, indexes, grad_outputs):
 class ClipGrad(function_node.FunctionNode):
 
     def __init__(self, x, x_min, x_max):
-        self.cond = (x_min <= x) * (x <= x_max)
+        self.cond = True
```

Comment:
How about asserting above this line that not both of min and max are `None` since it's an assumption that this class makes.